shared/src/types/api/question.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { DifficultyLevel, Language } from "../base";
+
+export class GetQuestionReq {
+  difficultyLevel: DifficultyLevel;
+}
```

Comment:
We probably don't need this since the room service will be the one invoking the question service to get a random question ID.

PS: `GET` requests can't/shouldn't have a body.